sorbet/rbi/gems/sorbet-static-and-runtime@0.5.9757.rbi

```diff
@@ -0,0 +1,8 @@
+# typed: true
+
+# DO NOT EDIT MANUALLY
+# This is an autogenerated file for types exported from the `sorbet-static-and-runtime` gem.
+# Please instead update this file by running `bin/tapioca gem sorbet-static-and-runtime`.
+
+# THIS IS AN EMPTY RBI FILE.
+# see https://github.com/Shopify/tapioca/wiki/Manual-Gem-Requires
```

Comment:
We can update https://github.com/Shopify/tapioca/blob/507cdfdb12cffdd1912b74065fb6b426be0fb773/lib/tapioca/gemfile.rb#L142 and get rid of this one.